services/mapper.go

```diff
@@ -130,7 +130,7 @@ func TransferOps(tx *evmClient.LoadedTransaction, startIndex int) []*RosettaType
 	return ops
 }
 
-func FeeOps(tx *evmClient.LoadedTransaction) []*RosettaTypes.Operation {
+func FeeOps(tx *evmClient.LoadedTransaction, native_currency *RosettaTypes.Currency) []*RosettaTypes.Operation {
```

Comment:
this function is used by all EVM chains, modify the shared util function here especially the function args can potentially break all downstream dependents, usually EVM chain should implement it according to its own flavour

also all EVM and L2 use ETH as native token, we shouldn't only modify the shared util function for BNB